components/motor/gpio/motor_encoder.go

```diff
@@ -161,30 +158,36 @@ func newEncodedMotor(
 		em.flip = -1
 	}
 
+	if rdkutils.Float64AlmostEqual(em.ticksPerRotation, 0, 0.01) {
+		em.ticksPerRotation = 1
+	}
+
```

Comment:
[q] why does this need to use `Float64AlmostEqual` but `mc.TicksPerRotation` on line 64 can be directly compared to 0?

Reply:
good catch! thanks!